packages/visualizations/src/components/Map/utils.ts

```diff
@@ -2,23 +2,36 @@ import chroma from 'chroma-js';
 import turfBbox from '@turf/bbox';
 import maplibregl from 'maplibre-gl';
 import geoViewport from '@mapbox/geo-viewport';
-
-export const LIGHT_GREY = '#CBD2DB';
-export const DARK_GREY = '#515457';
-
-export const colorShapes = (geoJson, values, colorsScale, emptyValueColor) => {
+import type { FeatureCollection, Feature, Position, BBox } from 'geojson';
+import type { Scale } from 'chroma-js';
+import type { Color, ColorsScale } from '../types';
+import type {
+    ChoroplethDataValue,
+    ChoroplethFixedTooltipDescription,
+    MapRenderTooltipFunction,
+} from './types';
+
+export const LIGHT_GREY: Color = '#CBD2DB';
+export const DARK_GREY: Color = '#515457';
+
+export const colorShapes = (
+    geoJson: FeatureCollection,
+    values: ChoroplethDataValue[],
+    colorsScale: ColorsScale,
+    emptyValueColor: Color
+) => {
```

Comment:
Since we will (maybe?) enforce a stricter typing checking on all packages, I'm reviewing this PR using a TS configuration similar to the one on the platform. In this context, the compiler complains because of the rule [`explicit-module-boundary-types`](https://github.com/typescript-eslint/typescript-eslint/blob/v4.31.2/packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md) (same thing line 143).

But that's a problem for an other day ^^ I'm completely ok to leave it as it is. But if you want to change it:

```TypeScript
{
    geoJson: FeatureCollection,
    bounds: {
        min: number,
        max: number
    }
}
```

Reply:
I didn't even know about that rule in the platform 😳 

I fixed it, better take good habits from the start!